web/frontend_initial_setup/src/pages/CalendarSch/index.jsx

```diff
@@ -0,0 +1,33 @@
+import { connect } from "react-redux";
```

Comment:
Let's change the name of the directory from `CalendarSch` to `CalendarSchedule` under this file